Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -176,6 +181,40 @@ define([
         return true;
     };
 
+    function getDiffuseUniformName(gltf) {
```

Comment:
This could be true... but to keep the code simple I'm not supporting this case right now.